### PR TITLE
Add initial BuildJSONController for build details

### PR DESCRIPTION
**Pull request by KrauseFx**
screen shot 2018-05-23 at 3 30 31 pm

Fixes https://github.com/fastlane/ci/issues/917

### CODE REVIEW

File: app/features-json/view_models/project_view_model.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require_relative "../../shared/json_convertible"
 
 module FastlaneCI
-  # View model to expose the basic info about a project.
+  # View model to expose the detailed info about a project.
```
Comment (KrauseFx) [inline app/features-json/view_models/project_view_model.rb]: @nakhbari can you confirm this is accurate? This and another class had the same class level description, and I wasn't 100% certain when which one is used

Comment (nakhbari) [reply app/features-json/view_models/project_view_model.rb]: yep this is good 👍

File: app/features-json/view_models/build_view_model.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    attr_reader :build_tools
+
+    # @return [String] The git URL
+    attr_reader :clone_url
+
+    # @return [String] The git branch
+    attr_reader :branch
+
+    # @return [String] The git ref
+    attr_reader :ref
```
Comment (nakhbari) [inline app/features-json/view_models/build_view_model.rb]: @KrauseFx what's the git ref?

Comment (KrauseFx) [reply app/features-json/view_models/build_view_model.rb]: We have some more docs on this over here https://github.com/fastlane/ci/blob/master/app/shared/models/git_fork_config.rb#L19-L21

I don't know for sure when this gets filled out, as every `ref` can also be represented by a `sha`. Maybe @taquitos knows more, as he invented the class `GitForkConfig`

**Pull request merged**